src/web/client/test/integration/errorHandler.test.ts

```diff
@@ -327,81 +249,18 @@ describe("errorHandler", () => {
     it("checkMandatoryPathParameters_whenEveryParameterIsPassedOrValid_shouldReturnTrue", () => {
         //Act
         const appName = "portal";
-        const entity = "entity";
-        const entityId = "512e50bd-a9d1-44c0-ba3f-5dc7f46e7216";
         //Action
-        const result = checkMandatoryPathParameters(appName, entity, entityId);
+        const result = checkMandatoryPathParameters(appName);
         //Assert
         expect(result).true;
     });
 
-    it("checkMandatoryPathParameters_whenEntityIsBlank_shouldReturnFalse", () => {
```

Comment:
Accordingly these test cases will be enhanced for new scenarios -do not simply remove the old test cases without adding coverage for new scenarios

Reply:
So for now these testcases are not required as if we don't have entity and entityid in path params, we try calling from query params, and if not there also we open the portal with no default page for multifile

Reply:
as per discussion, will raise a subsequent PR for UT, else this will become a huge PR for review.